akka-http/src/main/scala/akka/http/scaladsl/server/directives/PathDirectives.scala

```diff
@@ -177,13 +177,11 @@ trait PathDirectives extends PathMatchers with ImplicitPathMatcherConstruction w
   /**
    * If the request path ends with a slash, redirect to the same uri without trailing slash in the path.
    *
-   * '''Caveat''': [[pathSingleSlash]] directive will not match inside of this directive.
```

Comment:
I think the cave at is still valid. It's just a reminder not to expect trailing slashes inside that directive.